optuna/study/study.py

```diff
@@ -223,6 +227,9 @@ def get_trials(
 
         The returned trials are ordered by trial number.
 
+        .. seealso::
+            See :attr:`~optuna.study.Study.trials` for related method.
```

Comment:
Sorry, one more thing. Technically, property sounds more precise than `method`.

```suggestion
            See :attr:`~optuna.study.Study.trials` for related property.
```